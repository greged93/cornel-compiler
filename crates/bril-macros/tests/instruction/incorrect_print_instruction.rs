#![no_main]

use bril_macros::instruction;

instruction!(op = print, args = [a], value = 1);
