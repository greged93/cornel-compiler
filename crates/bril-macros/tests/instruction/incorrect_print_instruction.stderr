error: invalid instruction
 --> tests/instruction/incorrect_print_instruction.rs:5:1
  |
5 | instruction!(op = print, args = [a], value = 1);
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `instruction` (in Nightly builds, run with -Z macro-backtrace for more info)
